apps/web/src/pages/seats/index.tsx

```
@@ -1,14 +1,23 @@
+import ErrorBoundary from "@src/components/commons/ErrorBoundary";
+import { Suspense } from "react";
+
+import Error from "./Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)